docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+
+## From Maistra to Istio
+
+While OpenShift Service Mesh 1 and 2 releases were based on Istio, they included additional functionality that was maintained as part of the midstream Maistra project, which itself was based on the upstream Istio project. Maistra maintained several features that were not part of the upstream Istio project. While this provided extra features to OpenShift Service Mesh users, the effort to maintain the Maistra meant that OpenShift Service Mesh was usually several releases behind Istio with support omitted for major features like multi-cluster. Meanwhile, Istio has matured to cover most of the use cases addressed by Maistra. Basing OpenShift Service Mesh directly on Istio ensures that it will support users on the latest stable Istio features while Red Hat is able to contributing directly to the Istio community on behalf of its customers. 
+
+## The OpenShift Service Mesh 3 operator
+
+OpenShift Service Mesh 3 uses an operator that is maintained upstream as the sail-operator in the istio-ecosystem organization. This operator is smaller in scope and includes significant changes from the operator used in OpenShift Service Mesh 2 that was maintained as part of the Maisra.io project.
+
+## Observability integrations rather than addons
+
+A significant change in OpenShift Service Mesh 3 compared to OpenShift Service Mesh 2 is that the operator no longer installs and manages observability components such as Prometheus and Grafana with the service mesh control plane. It also no longer installs and manages distributed tracing components such as Tempo and OpenTelemetry (Jaeger and Elasticsearch in the past) or the Kiali dashboard. 
+
+The OpenShift Service Mesh 3 operator limits its scope to Istio related resources, with observability components supported and managed by the independent operators that make up [Red Hat OpenShift Observability](https://docs.redhat.com/en/documentation/openshift_container_platform/#Observability) such as [logging](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/logging/index), user workload [monitoring](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/monitoring/index), [distributed tracing](https://docs.redhat.com/en/documentation/openshift_container_platform/#/html/distributed_tracing/index). The [Kiali dashboard](https://docs.openshift.com/service-mesh/3.0.0tp1/kiali/ossm-kiali-assembly.html) along with OpenShift Service Mesh Console will continue to be supported with the Kiali operator.
```

Comment:
The URLs do not have OCP version. Is this intentional? 

The side-effect is that it does not land on the page you probably wanted it to land. 
i.e., for logging, the working URL is 
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/logging/index
whereas if I click on the link from this doc, it takes me to 
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17#/html/logging/index

Reply:
It was an attempt to make the links version agnostic, but I don't think that feature works with Red Hat docs yet, so will have to hard code to OCP 4.17 for now. I will let docs determine the best link for publishing.